### ISSUE TITLE
netconvert OpenDrive import misses junction connection

### ISSUE THREAD

**Issue by m-kro** [eclipse/sumo#4812]
The given OpenDrive file models a four-leg junction with turning options right/straight/left for all incoming lanes. However the netconvert import result shows no straight connection from top incoming edge to bottom outgoing edge. For other approaches, the turnings have been created successfully.
[Test_Variante_3_Strassen.zip](https://github.com/eclipse/sumo/files/2595109/Test_Variante_3_Strassen.zip)

**Comment by namdre** [comment]
I'm not sure if the solution covers all cases. Please let me know if you find further oddities.
May the example input be used in our public test suite?

**Comment by m-kro** [comment]
Yes, you may add the example to the tests. Currently, we have a couple of intersections to convert. If we observe again some missing connection, we will report it here. Thank you for your help!

**Comment by m-kro** [comment]
There are more issues with the OpenDrive import when using the additional options **--opendrive.import-all-lanes true --opendrive.internal-shapes true**. Below you see he same intersection as before, but: 
- More connections than expected (see the parallel lines right-left, bottom-top)
- Intersection geometry does not match connection length
- Odd connection geometry
![grafik](https://user-images.githubusercontent.com/33294728/49150700-fe37e300-f30d-11e8-8313-f8c81d82c145.png)

Then there is another intersection where the same comments apply **and** a connection is missing (only with the extra OpenDrive import options):
![grafik](https://user-images.githubusercontent.com/33294728/49150980-bcf40300-f30e-11e8-93e9-8a52ac2a7880.png)
Here is the OpenDrive file for it: 
[Test_Testkreuzung_mitSperrflaeche.zip](https://github.com/eclipse/sumo/files/2624400/Test_Testkreuzung_mitSperrflaeche.zip)

@namdre: Please correct the contribution information for the test to Technische Universität Braunschweig. Thank you.

**Comment by namdre** [comment]
The connectivity is correct but the shape of the internal lane is wrong. It's on top of the reverse direction internal lane and thus hidden.

**Comment by namdre** [comment]
@m-kro There are still issues with variable width lanes of type 'none' that affect the shape of the internal lanes but are currently treated as constant width.

**Comment by sapirbm** [comment]
hello @namdre 
**netconvert still misses connections.** in more than one junction in the files attached.
for example, in junction 167, in opendrive we can see 16 connections while in sumo only 12. netconvert dropped all of the right-lane-straight-line from all 4 junction directions. (roads 175, 178, 194, 197)
these roads and connections are missing in the net.xml file.

[maps.zip](https://github.com/eclipse/sumo/files/2848707/maps.zip)
